arches/app/models/graph.py

```diff
@@ -563,19 +564,18 @@ def delete(self):
                 )
             )
 
-    def delete_instances(self, verbose=False):
+    def delete_instances(self, userid={}, verbose=False):
```

Comment:
Seems like None would be better here. I was suggesting instead that the `else None` in `delete_resources` could be `else {}` until we do the follow-up work.

Reply:
fix in latest